app/background.html

```diff
@@ -6,7 +6,7 @@
   <body>
     <script src="./globalthis.js" type="text/javascript" charset="utf-8"></script>
     <script src="./sentry-install.js" type="text/javascript" charset="utf-8"></script>
-    {{@if(it.useLavamoat)}}
+    {{@if(it.applyLavaMoat)}}
```

Comment:
We liked `applyLavaMoat` better than `useLavamoat`, which was improperly cased in any event.